roles/common/tasks/debian.yml

```diff
@@ -212,3 +212,22 @@
       - unzip
       - curl
   tags: package
+
+- set_fact:
```

Comment:
We should not be worry about python 2 specially. We should be looking for runtime param of `ansible_python_interpreter` It's not just matter of Python 2 vs 3 but there can be multiple versions of python on managed nodes, and user may want to use one of them for many reasons.

Reply:
Let’s discuss this, 
As far as I understood, `ansible_python_version` is another runtime param which we have used now to detect the selected python version, 
ansible_python_interpreter is set to auto (default), auto_legacy, auto_legacy_silent, auto_silent
References - 
https://sleeplessbeastie.eu/2020/02/07/how-to-instruct-ansible-to-use-specific-version-of-python/
https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html